jobclient/src/test/java/com/twosigma/cook/jobclient/JobTest.java

```diff
@@ -38,6 +39,11 @@
      */
     private Job _initializedJob;
 
+    /**
+     * A constraint will be applied to the job.
+     */
+    private TaskConstraint.Constraint _JobConstraint;
```

Comment:
Should we also add tests in `cook.test.jobclient.jobclient`?

Reply:
Added.